nvtabular/inference/triton/model_hugectr.py

```diff
@@ -83,16 +83,17 @@ def execute(self, requests: List[InferenceRequest]) -> List[InferenceResponse]:
                 output_tensors.append(Tensor("DES", np.array([0])))
 
             if "cats" in self.column_types:
+                cats_np = _convert_cudf2numpy(output_df[self.column_types["cats"]], np.uint32)
                 output_tensors.append(
                     Tensor(
                         "CATCOLUMN",
-                        _convert_cudf2numpy(output_df[self.column_types["cats"]], np.uint32),
+                        cats_np,
                     )
                 )
             else:
                 output_tensors.append(Tensor("CATCOLUMN", np.array([0])))
 
-            row_index = [i for i in range(len(self.workflow.column_group.input_column_names))]
+            row_index = [i for i in range(len(cats_np))]
             row_index = np.array(row_index).astype(np.int32)
```

Comment:
We can do this without creating a python list in pure numpy 
```suggestion
            row_index = np.arange(len(cats_np), dtype=np.int32)
```